### DISCUSSION TITLE
Is there any settings where we can set max request payload?

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2991] [Archive]
I have the error `request entity too large`, is there any env I can set to adjust that?

**Comment by rijkvanzanten** [comment]
I don't think this is thrown from Directus. Check your web servers' settings!

**Reply by botzill** [reply]
Well, I did increase nginx settings for that but I still get this error. I was wondering maybe it's related to nodejs.

**Reply by botzill** [reply]
Example expressjs has this kind of limit: https://github.com/expressjs/body-parser#limit

**Reply by rijkvanzanten** [reply]
Ah! What was the payload you're sending (if you don't mind me asking)?

Also, yes lets make that configurable

**Comment by rijkvanzanten** [comment]
This will be implemented. Keep an eye on https://github.com/directus/next/issues/1005 for more information

**Reply by botzill** [reply]
OK, thx.
